h2o-algos/src/main/java/hex/gam/GAM.java

```diff
@@ -31,7 +35,15 @@
 
 
 public class GAM extends ModelBuilder<GAMModel, GAMModel.GAMParameters, GAMModel.GAMModelOutput> {
-
+  double[][] _knots;
+  boolean _cv_on = false; // set to true if cross-validation is enabled
```

Comment:
Do we have any other internal rule for using an underscore in Java code than at the beginning of a parameter? Because the standard is to use CamelCase. It is not clear to have two different styles in our Java code. 

```suggestion
  boolean _cvOn = false; // set to true if cross-validation is enabled
```

Reply:
I think _cvOn is a good idea.  I changed it.